### PR TITLE
Error fixed during "make" by adding cv::

### PR DESCRIPTION
**Pull request by SamVadidar**
### This pullrequest changes
a function call to let the  "make" work with no error!

### CODE REVIEW

File: modules/cvv/src/qtutil/filter/diffFilterWidget.cpp [hunk 1]
```diff
@@ -69,8 +69,8 @@ void DiffFilterFunction::applyFilter(InputArray in, OutputArray out) const
 	}
 
 	cv::Mat originalHSV, filteredHSV;
-	cv::cvtColor(in.at(0).get(), originalHSV, COLOR_BGR2HSV);
-	cv::cvtColor(in.at(1).get(), filteredHSV, COLOR_BGR2HSV);
```
Comment (alalek) [inline modules/cvv/src/qtutil/filter/diffFilterWidget.cpp]: What is used version of OpenCV?

`using namespace cv;` on line 16 should fix that properly.

**Comment by SamVadidar** [comment]
I just added cv:: to COLOR_BGR2HSV and it worked! I wanted to change it for everyone.
Cheers,sam
    On Monday, April 15, 2019, 2:22:42 PM GMT+2, Alexander Alekhin <notifications@github.com> wrote:  
 
 
@alalek requested changes on this pull request.

In modules/cvv/src/qtutil/filter/diffFilterWidget.cpp:
> @@ -69,8 +69,8 @@ void DiffFilterFunction::applyFilter(InputArray in, OutputArray out) const
 	}
 
 	cv::Mat originalHSV, filteredHSV;
-	cv::cvtColor(in.at(0).get(), originalHSV, COLOR_BGR2HSV);
-	cv::cvtColor(in.at(1).get(), filteredHSV, COLOR_BGR2HSV);

What is used version of OpenCV?

using namespace cv; on line 16 should fix that properly.

**Comment by alalek** [comment]
> added cv:: 
> I wanted to change it for everyone.

This is not needed and will be rejected.
HEAD of "3.4"/"master' branch is already fixed (#1905)

**Comment by SamVadidar** [comment]
If it was not needed, I wouldn't get an error during installation from source, I suppose!
    On Monday, April 15, 2019, 3:18:57 PM GMT+2, Alexander Alekhin <notifications@github.com> wrote:  
 
 

added cv::
I wanted to change it for everyone.


This is not needed and will be rejected.
HEAD of "3.4"/"master' branch is already fixed (#1905)

**Pull request closed without merging**